### ISSUE TITLE
Latest build does not sync

### ISSUE THREAD

**Issue by slozan0** [serras/SparkleShare#10]
Dear all,

I recently build the latest update in a win7 (32bit) machine without errors, however I'm not able to sync to the server. I tried the same installable in another machines with the same problem. I'm able to sync with the server using a 0.2.5 ubuntu and 0.4.0-131-ga4496cb win7 client. Any ideas where I should like to debug the problem?

**Comment by serras** [comment]
Sorry for the delay, these last days have been crazy for me. I'll look at the problem during the weekend.

**Comment by resnostyle** [comment]
I am having a similar issue. It works correctly if its a empty repo, but ones with any files wont make any progress. I left one running 24 hours and no progress..

Sparkleshare also says "not everything is synced".

Windows 7, 32bit. Using the installer.

**Comment by serras** [comment]
May you check with the latest version in the repo? I merged a bunch of commits from the main repo.

**Comment by slozan0** [comment]
HI @serras I just updated and compiled without error. This time I deleted the shared folders and tried to add a folder with some data and a project without data both failed with "Something went wrong". Let me know if you need logs in order to debug the problem

**Comment by serras** [comment]
Yep, I think logs will make it easier to spot the problem. Thanks for the
help.

2012/2/28 Saul Lozano <
reply@reply.github.com

**Comment by serras** [comment]
In my XP machine everything seems to work OK: I've created a brand-new Github repo to try and sync was perfect. Maybe there's a problem when using Win 7. Did you try every time with Win 7? May you try running the program with admin privileges?

**Comment by resnostyle** [comment]
I have it running now. I had to run it with Admin Privs. And then it
started working pretty well. Otherwise I got repeated error messages or
something wouldnt work.

On Tue, Feb 28, 2012 at 6:35 AM, Alejandro Serrano <
reply@reply.github.com

**Comment by resnostyle** [comment]
Im curious why doesnt sparkleshare use more bandwidth? It seems like it's barely using my nic.
